### DISCUSSION TITLE
关于m4s格式

### DISCUSSION THREAD

**Discussion by NoNova7** [lifegpc/bili#80] [Q&A] [answered]
大佬你好, 我想用ffmpeg合并多p的视频, 想请问问您, m4s格式更下倾向于file-level concatenation(mkv)还是non-file-level concatenation(mp4).
换而言之: 
non-file-level: ffmpeg -f concat -safe 0 -i mylist.txt -c copy output.mp4
file-level: ffmpeg -i "concat:input1|input2" -codec copy output.mkv
这两个方案哪种好.

谢谢

**Comment by lifegpc** [comment, marked as answer]
第一种
命令行调用出错概率会增加。而且过长的命令行会无法运行
